Source/NexusForever.AuthServer/Network/Message/Handler/AuthenticationHandler.cs

```diff
@@ -44,7 +44,7 @@ void SendServerAuthDenied(NpLoginResult result)
                 }
 
                 // TODO: might want to make this smarter in the future, eg: select a server the user has characters on
-                ServerInfo server = ServerManager.Instance.Servers.FirstOrDefault();
+                ServerInfo server = ServerManager.Instance.Servers.Where(s => s.IsOnline).FirstOrDefault();
```

Comment:
This change is good but could be shortened further by placing the predicate in the FIrstOrDefault call.
```suggestion
                ServerInfo server = ServerManager.Instance.Servers.FirstOrDefault(s => s.IsOnline);
```

Reply:
Good catch, modified